### PR TITLE
add AcceptanceTests.md

### PR DESCRIPTION
**Pull request by edwinduong**
Describes manual testing procedures for:
- Logging in
- Registration
- Logging out

### CODE REVIEW

File: docs/AcceptanceCriteria.md [hunk 1]
```diff
@@ -224,3 +224,55 @@ So I know which parking spot is being listed, auctioned, or both at a glance”
...
+3. Select the "First Name" text field, input "Robert"
+4. Select the "Last Name" text field, input "Guderian"
+5. Select the "Email" text field, input "robg@cs.umanitoba.ca"
+6. Select the "Password" text field, input "password"
+7. Select the "Confirm Password" text field, input "password"
+8. Press the "Sign Up" button
+*Expected: "Account created successfully" is displayed*
+
+**Test 2**
+1. Open Firefox, navigate to "http://localhost:5173/auth/login"
```
Comment (TanoVip123) [inline docs/AcceptanceCriteria.md:270]: Please change Firefox to browser to be more general

**Review by TanoVip123 — CHANGES_REQUESTED** [review]

File: docs/AcceptanceCriteria.md [hunk 2]
```diff
@@ -224,3 +224,55 @@ So I know which parking spot is being listed, auctioned, or both at a glance”
...
+**Test 2**
+1. Open your browser of choice, navigate to "http://localhost:5173/auth/login"
+2. Select the "Email" text field, input "robg@cs.umanitoba.ca"
+3. Select the "Password" text field, input "password"
+4. Press the "Login" button
+*Expected: Website redirects to a different page*
+5. Press the "my profile" button
+*Expected: "Robert Guderian" and "robg@cs.umanitoba.ca" are displayed*
+6. Press the "Logout" button
+*Expected: Website redirects to a the login page"
```
Comment (TanoVip123) [inline docs/AcceptanceCriteria.md:278]: currently the webpage is redirecting you to a welcome page if you logout so maybe change this from 'login page' to 'welcome page'

Comment (edwinduong) [reply docs/AcceptanceCriteria.md:278]:
```suggestion
*Expected: Website redirects to a different page"
```

Comment (edwinduong) [reply docs/AcceptanceCriteria.md]: Updated (also I noticed a spelling error)

**Review by TanoVip123 — APPROVED** [review]

**Pull request merged**